File: Practica1/Ej4/Cuenta.java
class Cuenta{

    private String correo, contrasena;

    public Cuenta(String cor, String con){
        this.correo=cor;
        this.contrasena=con;
    }

    public String getCorreo(){
       return correo; 
    }

    public String getContrasena(){
        return contrasena;
    }

    public void setCorreo(String c){
        this.correo=c;
    }

    public void setContrasena(String c){
        this.contrasena=c;
    }

}

File: Practica2/Ej1/lib/Cuenta.dart
class Cuenta{

  String _correo = "";
  String _contrasenia= "";

  Cuenta (String correo, String contrasenia){
    _contrasenia = contrasenia;
    _correo = correo;
  }

  String getCorreo(){
    return _correo;
  }

  String getContrasenia(){
    return _contrasenia;
  }

  void setCorreo(String c){
    _correo = c;
  }

  void setContrasenia (String c){
    _contrasenia = c;
  }

  //Devuelve la longitud del corre
  int lenCorreo(){
    return _correo.length;
  }

  //Devuelve la longitud de la conraseña
  int lenContrasenia(){
    return _contrasenia.length;
  }

}